script.js

```diff
@@ -9,10 +9,10 @@ $(function () {
         var blue = $(".blue .number").prev("input:checked").length;
         var strikes = $(".strikes").find("input:checked").length;
 
-        if ($(".red .lock + input:checked").length) red += 1;
-        if ($(".yellow .lock + input:checked").length) yellow += 1;
-        if ($(".green .lock + input:checked").length) green += 1;
-        if ($(".blue .lock + input:checked").length) blue += 1;
+        if ($("#red-lock").prop("checked")) {red += 1};
+        if ($("#yellow-lock").prop("checked")) yellow += 1;
+        if ($("#green-lock").prop("checked")) green += 1;
+        if ($("#blue-lock").prop("checked")) blue += 1;
```

Comment:
The `+` means immediate proceeding sibling, so this _was_ checking the input after the lock. With this change, it's now checking the lock's input for scoring. Lol, while the change is wrong, I wonder if the original code was wrong as well. Are these lines needed at all? I'd need to double-check the qwixx rules to be sure.